trace/src/main/org/jetbrains/lincheck/trace/TraceRecorderTracePoints.kt

```diff
@@ -27,18 +27,54 @@ import java.io.DataInput
 import java.io.DataOutput
 import java.math.BigDecimal
 import java.math.BigInteger
+import java.util.EnumSet
 import java.util.concurrent.atomic.AtomicInteger
+import kotlin.experimental.or
 import kotlin.reflect.KClass
 
 private val EVENT_ID_GENERATOR = AtomicInteger(0)
 
 var INJECTIONS_VOID_OBJECT: Any? = null
 
+/**
+ * Describes status of tracepoint in trace diff
+ */
+enum class DiffStatus {
+    /**
+     * Trace is not diff at all, or this tracepoint is identical in two traces.
+     */
+    COMMON,
```

Comment:
```suggestion
    NONE,
```
or could as well just use `DiffStatus?` with `null` denoting non-diff data.

Reply:
I'm leaving `COMMON` for diffs and enable `null` for simple traces.